portfolio/src/main/webapp/style.css

```diff
@@ -22,4 +22,36 @@ img {
 .comments {
   display: block;
   list-style: none;
+}
+
+.comment-submit {
+  padding: 5px 10px;
+}
+
+input[type=submit], button {
+  background-color: #FF6341;
+  border: none;
```

Comment:
this is a bit weird to me. if you're specifying `border: none`, then `border-radius` should be a no-op right? I feel like only one of these can be right.

Reply:
`border-radius` curves the edges of the div/button/element to be round instead of a 90 degree angle. `border` refers to the actual border styling like 'dotted', `3px` wide, `blue`, etc.

https://www.w3schools.com/cssref/css3_pr_border-radius.asp

Reply:
Ah, good point. You're absolutely right.